packages/calcite-components/src/components/slider/slider.tsx

```diff
@@ -93,6 +92,13 @@ export class Slider
   /** When `true`, indicates a histogram is present. */
   @Prop({ reflect: true, mutable: true }) hasHistogram = false;
 
+  /**
+   * Used to configure where the highlight is placed along the slider track.
+   *
+   * **Note**: range mode will always display range between min and max handles.
+   */
+  @Prop({ reflect: true }) highlightMode: "default" | "none" | "mirrored" = "default";
```

Comment:
comment: Is there a better name for "default"? Otherwise, default doesn't really mean anything other than its the default whereas the others have some meaning to them.

Reply:
> "start" (default) / "end" / "none" ? There is also the separate mirrored property on Slider which might be confusing.

<img src="https://i.giphy.com/h4TdHo3RExSbHd9bOe.webp">

Reply:
Went with `fillPlacement: "start" | "end" | "none"` since [`meter`](https://developers.arcgis.com/calcite-design-system/components/meter/) introduces fill-related props, which would apply to slider too.